index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <header>
+            <div class="logo">logos</div>
+            <div class="humbuger"><img src="images/icones/Union.png" alt="humburger"></div>
+        </header>
+        <div class="contain">
```

Comment:
- [ ] Kindly make sure this whole section has a container tag and it's a `<section>`, also ensure the content of the sections is realigned using flexbox.